docs/docsource/installation/prepare/configure_logical_volumes.rst

```diff
@@ -38,42 +31,198 @@ For each node, one volume of each of the following sizes:
+  ``fstype`` to ``ext4`` and ``mkfsOptions`` to ``'["-m", "0"]'``, indenting as
+  shown in the example.
+* **provisioner:** Define this as ``kubernetes.io/no-provisioner``.
+* **reclaimPolicy:** Define this as ``Retain``.
+* **volumeBindingMode:** Define this as ``WaitForFirstConsumer``.
+
+#. Create a storageclasses.yaml file based on the following example. For the
+   sizing described in :ref:`Minimum Volume Size for Cluster Deployments`,
+   you can copy, paste, and save this example.
+
+   ::
+
+    ---
+    apiVersion: storage.k8s.io/v1
+    kind: StorageClass
+    metadata:
+      name: zenko-storage
+      labels:
+        zenko: storageclass
+      annotations:
+        storageclass.kubernetes.io/is-default-class: "true"
+    mountOptions:
+      - rw
+      - discard
+    parameters:
+      fsType: ext4
+      mkfsOptions: '["-m", "0"]'
+    provisioner: kubernetes.io/no-provisioner
+    reclaimPolicy: Retain
+    volumeBindingMode: WaitForFirstConsumer
+
+#. Run ``kubectl apply -f storageclasses.yaml`` to create the resources.
+
+   ::
+      
+      storageclass.storage.k8s.io/zenko-storage created
+
+#. Run ``kubectl get storageclasses`` to verify that the storage class definitions have
+   been accepted.
+
+   ::
+
+      NAME                      PROVISIONER                    AGE
+      metalk8s-prometheus       kubernetes.io/no-provisioner   80m
+      zenko-storage (default)   kubernetes.io/no-provisioner   52s
+
+Set Up Logical Volumes with MetalK8s
+------------------------------------
+
+Use the MetalK8s GUI to set volume sizes. You *must* size the volumes during
+setup. The easiest way to do this is to set the volume sizes when creating them,
+then to assign them as follows:
+
+#. Open the MetalK8s Platform user interface at [bootstrap]:8443.
+
+#. Click the **Nodes** tab.
+
+#. Review the Nodes list for nodes in your cluster with a **Deploy** button
+   available in the **Deployment** column. Click the **Deploy** button to deploy
+   available nodes.
+
+For each node,
+
+#. Click the name of the node in the Platform menu to display node details.
+
+   .. image:: ../Graphics/MK8s_node_select.png
+
+#. Click the **Volumes** tab.
+
+   .. image:: ../Graphics/MK8s_details.png
+
+#. Click the **+** button.
+
+   .. image:: ../Graphics/MK8s_volume_tab.png
+
+#. The **Create a New Volume** window displays.
+
+   .. image:: ../Graphics/MK8s_volume_create.png
+
+#. Enter the volume's
+
+   a. **Name** This must be AWS S3 compliant: lowercase letters, numbers, and
+      hyphens only.
+   #. **Labels** This is an optional, customizable key-value metadata pair for
+      objects that pass through this volume.
+   #. **Storage Class** This field contains a pull-down menu of the storage
+      class options you created in the Storage Class Manifest
+      (storageClasses.yaml) in :ref:`Create a Storage Class Manifest`.
+   #. **Type** This presents two options: **RawBlockDevice** and
```

Comment:
I understand what the "Type" is about, but I don't see it described above in the Storage Class Manifest section, so wondering if it can be referred to here.

As an extra warning it could be useful to stress that `SparseLoopDevice` is meant for testing environments only (performance degradation expected).

Reply:
Removed the reference to the manifest (the sample manifest changed, the reference hadn't, so good spot!) 

Added a sentence about test-only/performance degradation.